src/pcapi/core/users/api.py

```diff
@@ -888,3 +890,21 @@ def fraud_manager(user: User, phone_number: str) -> typing.Generator:
     except exceptions.PhoneValidationAttemptsLimitReached as error:
         fraud_api.handle_phone_validation_attempts_limit_reached(user, error.attempts)
         raise
+
+
+def update_last_connection_date(user):
+    previous_connection_date = user.lastConnectionDate
+    last_connection_date = datetime.now()
+
+    if not previous_connection_date or last_connection_date - previous_connection_date > timedelta(minutes=15):
+        user.lastConnectionDate = last_connection_date
+        repository.save(user)
+
+        if not previous_connection_date or datetime.combine(last_connection_date, time(0, 0)) - datetime.combine(
+            previous_connection_date, time(0, 0)
+        ) >= timedelta(days=1):
```

Comment:
Variante (un peu plus concise), qui me semble équivalente :

```suggestion
        if not previous_connection_date or last_connection_date.date() - previous_connection_date.date() >= timedelta(days=1):
```

Reply:
mieux :)